packages/app-project/src/components/AuthModal/AuthModal.js

```diff
@@ -0,0 +1,61 @@
+import { Modal, Tabs, Tab, withThemeContext } from '@zooniverse/react-components'
+import { Box } from 'grommet'
+import { func, string } from 'prop-types'
+import React from 'react'
+import styled from 'styled-components'
+
+import authModalTheme from './theme'
+import LoginForm from './components/LoginForm'
+import withOnlyRenderOnBrowser from './shared/components/withOnlyRenderOnBrowser'
```

Comment:
This import hasn't been switch to the one moved into lib-react-components, so we have duplicate code now.

Reply:
Fixed via #888